worlds/generic/docs/commands_en.md

```diff
@@ -26,7 +26,7 @@ including the exclamation point.
 ### Utilities
 - `!countdown <number of seconds>` Starts a countdown using the given seconds value. Useful for synchronizing starts.
   Defaults to 10 seconds if no argument is provided.
-- `!alias <alias>` Sets your alias, which allows you to use commands with the alias rather than your provided name.
+- `!alias <alias>` Sets your alias, which allows you to use commands with the alias rather than your provided name. `!alias` on it's own will reset the alias to the player's original name.
```

Comment:
While this document does currently break the rule, and it's out of scope for this PR, our style guide does specify lines should only be 120 characters. Also, possessive it has no `'` :)
```suggestion
- `!alias <alias>` Sets your alias, which allows you to use commands with the alias rather than your provided name.
  `!alias` on its own will reset the alias to the player's original name.
```